tests/browser/features/happy.feature

```diff
@@ -21,5 +21,9 @@ Feature: Happy path
     Then they should see the answer security questions page
     When they continue from answer security questions
     Then they should see the enter-4-digits-bank-account-tax-credits question page
-    When they enter amount and continue from the enter-4-digits-bank-account-tax-credits question page
+    When they do not enter an account number and continue from the enter-4-digits-bank-account-tax-credits question page
+    Then they should see enter account number error message
+    When they enter an invalid account number and continue from the enter-4-digits-bank-account-tax-credits question page
+    Then they should see enter correct account number error message
+    When they enter correct account number and continue from the enter-4-digits-bank-account-tax-credits question page
```

Comment:
These are really unhappy path scenarios, could we pull out to a separate feature like https://github.com/govuk-one-login/ipv-cri-kbv-front/blob/main/test/browser/features/validation.feature

Reply:
Discussing with @AmritSidhu we realised we can't run a validation feature separately yet due to a limitation in the way the imposter mock is set up/run - we've raised a ticket [PYIC-5705](https://govukverify.atlassian.net/browse/PYIC-5705) to look at this.

For now let's keep the validation scenarios in here